README.md

```diff
@@ -1,2 +1,66 @@
-# dev-standards
-Documentation for all the development category standards followed at Osmosys
+# Osmosys Dev Standards
+
+Welcome to the Osmosys's Dev Standards Repository! This open-source project is a cornerstone of our commitment to excellence in software development. We've curated an extensive collection of coding standards, CI setup guides, and best practices spanning a multitude of programming languages. By collaborating here, you're contributing to the development community's efforts to streamline workflows, improve code quality, and promote best practices.
+
+## Table of Contents
+
+- [Osmosys Dev Standards](#osmosys-dev-standards)
+  - [Table of Contents](#table-of-contents)
+  - [Overview](#overview)
+  - [Languages Covered](#languages-covered)
+  - [Getting Started](#getting-started)
+  - [Contributing](#contributing)
+  - [Join Us!](#join-us)
+
+## Overview
+
+The Dev Standards Repository is a collection of detailed guides, each catering to a specific programming language or technology. Our mission is to create a hub of knowledge that empowers developers to follow consistent coding standards, set up efficient workflows, and embrace best practices.
+
+## Languages Covered
+
+We provide guides for a diverse array of languages and technologies, including but not limited to:
+
+- Power Apps
```

Comment:
Can we make them links?

Reply:
Also for now only keep the following

- These are the only ones which we published properly other we still have to work on them

- UI Guidelines
- Angular
- CSS
- Database
- Dotnet
- Flutter
- HTML
- Log Format
- PHP

Reply:
Yes, it's a good idea to make them links.
I have updated the list and added corresponding links to them